azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/RuntimeStackUtils.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.webapp.utils;
+
+import com.microsoft.azure.management.appservice.RuntimeStack;
+import org.codehaus.plexus.util.StringUtils;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class RuntimeStackUtils {
+
+    public static final String DEFAULT_JAVA_VERSION = "java11";
```

Comment:
name conflict of DEFAULT_JAVA_VERSION = JavaVersion.JAVA_8_NEWEST;

Reply:
Change name to default linux java version and set both windows and linux default value to java11